public/locales/en/app.ftl

```diff
@@ -0,0 +1,7 @@
+support = Support
+
+feedback = Give Feedback
+
+terms = Terms&nbsp;and&nbsp;Privacy
+
+copyright = Portions of this content are © 1998-2018 by individual mozilla.org contributors. Content available under a Creative Commons license
```

Comment:
Actual markup below is:

```html
<p data-l10n-id="copyright">
  Portions of this content are &copy; 1998-2018 by individual mozilla.org contributors.
  <br />
  Content available under a <a {{> analytics/link_event eventLabel="Creative Commons"}} href="https://www.mozilla.org/foundation/licensing/website-content/" target="_blank" rel="noopener">Creative Commons license</a>.
</p>
```

Not sure if the .ftl files should use &amp;copy; entity or if we need anything special for link preserving.